### DISCUSSION TITLE
How to share your video-conference window among attendees – or, the many ways of splitting a rectangle in many

### DISCUSSION THREAD

**Discussion by deleolajide** [igniterealtime/openfire-pade-plugin#200] [Ideas]
https://www.alfonsobeato.net/math/the-many-ways-of-splitting-a-rectangle-in-many/

**Comment by gjaekel** [comment]
Do you plan to improve the tile view? I also though about this, e.g. to give at least the dominant speaker (or the firstN) a bigger plane.

The paper you cite isn't of use, because here we have rectangles with fixed aspect ratio. But instead of building a simple N x N matrix, one can M stack stripes of rectangles with a rising N: Show two participants on first, 3 on next, 4 on next and so on.

**Comment by deleolajide** [comment]
No plans at the moment. Just a place holder to discuss how we can improve layout and reduce video conferencing fatigue
